event/models.py

```diff
@@ -1,3 +1,12 @@
 from django.db import models
 
 # Create your models here.
+class Event(models.Model):
+    title = models.CharField(max_length=200)
+    description = models.TextField()
+    image = models.ImageField(upload_to='event_images/', null=True, blank=True)
+    date = models.DateTimeField()
+    location = models.CharField(max_length=200)
```

Comment:
What does location mean here, is it part of the data of an event from the UI?

Reply:
The event venue/address. I will change it from location to venue_address